packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -74,13 +74,9 @@ describe('ApmServer', function() {
   var apmServer
   var configService
   var loggingService
-  var originalTimeout
   var performanceMonitoring
 
   beforeEach(function() {
-    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000
```

Comment:
I think these were here because we make actual connection to the APM server and that can take longer than the default timeout interval

Reply:
reverted and added a comment. Also, we don't do this on performance-monitoring for example so it might not be an issue at all.